app/src/renderer/components/staking/ModalStake.vue

```diff
@@ -0,0 +1,112 @@
+<template lang="pug">
+  .modal-stake
+    .stake-header
+      img.icon(class='stake-atom' src="~assets/images/cosmos-logo.png")
+      span.tm-modal-title Stake
+
+    tm-form-group.stake-form-group(
+      :error='$v.amount.$invalid'
+      field-id='amount'
+      field-label='Amount'
+    )
+      tm-field#amount(
+        type="number"
```

Comment:
let's add the following here as well:
```
:max="this.maximum"
:min="0"
v-focus
```

Reply:
(check [here](https://github.com/cosmos/voyager/pull/1310/files#diff-83b38b5ba2b68f6f0e53f186c743d3b8) for how to add the `v-focus` )

Reply:
Wow, that's a great improvement to add the min and max to the field!